_test/test_IX_classes/test_plot_singleton.m

```diff
@@ -2,16 +2,22 @@
     % Test genieplot singleton
     %
     % Tests the setting, getting and resetting of the properties
-    
+    %
+    % In addition, tests the interoperability between the two different syntaxes
+    % for use of genieplot.
+
     properties
         S_default       % Structure with default property values
+        % store current graphics settings in the structure back to revert it back
```

Comment:
Unnecessary repetition of "back"; also, the graphics settings will be stored in the object property, not a structure? Change:
"Store current graphics settings in the structure back to revert it back..."
to
"Store current graphics settings so they can be to reverted..."